### PR TITLE
docs: integrate iOS SDK

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

integration guide for iOS SDK

## Linear Issue Reference

LOG-2156

## Testing

image

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]
LGTM

File: packages/docs/docs/tutorial/integrate-sdk/ios.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+sidebar_label: iOS
+---
+
+import Step from '@components/Step';
```
Comment (xiaoyijun) [inline packages/docs/docs/tutorial/integrate-sdk/ios.mdx:5]: It seems that this `Step` is unused.

Comment (demonzoo) [reply packages/docs/docs/tutorial/integrate-sdk/ios.mdx:5]: Yes, we talked about it last night and we decided to separate the usages in AC and docs apart, which means AC will not try to re-use the mdx files in docs.
Hence the `Step` should be removed from this docs mdx file, and I will make a copy of this in AC but with `Step`

**Review by xiaoyijun — APPROVED** [review]
Overall LGTM

**Pull request merged**